D/difmap/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "difmap"
+version = v"2.5.5"
+
+sources = [
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/difmap/difmap2.5e.tar.gz", "457cd77c146e22b5332403c19b29485388a863ec494fff87137176396fc6a9ff"),
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4")
```

Comment:
Can we build pgplot in its own package?

Reply:
Below I set pretty specific (possibly?) options for pgplot, otherwise I couldn't compile it for some reason. So if this is fine for a separate pgplot package then I can split it.

Reply:
> Below I set pretty specific (likely) options for pgplot, otherwise I couldn't compile it for some reason. So if this is fine for a separate pgplot package then I can split it.

I don't understand what's the problem with splitting pgplot into its own package.  I believe we should do that.